trpcultivate_phenotypes/src/Plugin/Validators/EmptyCell.php

```diff
@@ -52,58 +40,62 @@ public static function create(ContainerInterface $container, array $configuratio
    * Validate the values within the cells of this row.
    *
    * @param array $row_values
-   *   The contents of the file's row where each value within a cell is
-   *   stored as an array element.
+   *   An array of values from a single row/line in the file where each value
+   *   is a single column.
    *
    * @return array
    *   An associative array with the following keys.
-   *   - case: a developer focused string describing the case checked.
-   *   - valid: FALSE if any of the cells being checked are empty and TRUE otherwise.
-   *   - failedItems: an array of "items" that failed with the following keys, to
-   *     be used in the message to the user. This is an empty array if the data row input was valid.
-   *     - empty_indices: A list of indices which were checked and found to be empty
+   *   - 'case': a developer focused string describing the case checked.
```

Comment:
Add the hyphen to match the previous validator.
```suggestion
   *   - 'case': a developer-focused string describing the case checked.
```

Reply:
Apparently DuplicateTraits was the only one using the hyphen 😅 All validators are now consistent with it as of commit [1a62664](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/1a6266446f70224d05194ea6085b256fa0754a16)!